examples/acceptFromIDs.js

```diff
@@ -1,6 +1,16 @@
-var SteamUser = require('steam-user');
-var client = new SteamUser();
-var TradeOfferManager = require('steam-tradeoffer-manager');
+const SteamUser = require('steam-user'),
```

Comment:
i think requiring should be const cuz it will never change, the vars even if not changed, will be used whilst the requires are just going into the vars and wont be used (hard to explain)